plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -148,6 +148,23 @@ export class EventPipelineRunner {
             event = normalizeProcessPerson(event, processPerson)
         }
 
+        if (event.event === '$$client_ingestion_warning') {
+            const warningAck = captureIngestionWarning(
+                this.hub.db.kafkaProducer,
+                event.team_id,
+                'client_ingestion_warning',
+                {
+                    eventUuid: event.uuid,
+                    event: event.event,
+                    distinctId: event.distinct_id,
+                    message: event.properties?.$$client_ingestion_warning_message,
+                },
+                { alwaysSend: true }
+            )
+
+            return this.registerLastStep('clientIngestionWarning', [event], [warningAck])
```

Comment:
Not super familiar with this code - I mostly copied it from the above example. Is the [event] part correct?